test/minigzip.c

```diff
@@ -276,6 +276,12 @@ int main(int argc, char *argv[]) {
     char *level = "6";
     char *type = "b";
 
+
+    if (argc <= 1) {
```

Comment:
argc cannot be 0, it's always at least 1, as the process name is the first argument.

Reply:
Fixed.